### DISCUSSION TITLE
custom classregex autocompletion

### DISCUSSION THREAD

**Discussion by sillymasha** [unocss/unocss#4255] [Q&A]
eg in tailwind we have `"tailwindCSS.experimental.classRegex"`
how to setup this in uno?

**Comment by zyyv** [comment]
Refer to https://unocss.dev/tools/autocomplete

**Reply by sillymasha** [reply]
still doesnt work or i just dont understand, tried the following to make autocompletion work in dioxus:
```
rules: [
  [
    /class: "(.*)"/,
  ],
]
```
